litmus-portal/frontend/src/theme/index.tsx

```diff
@@ -7,29 +7,29 @@ import Image from '../icons/arrow.png';
 declare module '@material-ui/core/styles/createMuiTheme' {
   interface Theme {
     // newProperty: {
-    // 	key: value;
-    // }
+    //   key: value;
+    // };
   }
   // allow configuration using `createMuiTheme`
   interface ThemeOptions {
     // newProperty?: {
-    // 	key?: value;
-    // }
+    //   key?: value;
+    // };
   }
 }
 
 // Augument the Palette interface
 declare module '@material-ui/core/styles/createPalette' {
   interface Palette {
     // newProperty: {
-    // 	key: value;
-    // }
+    //   key: value;
```

Comment:
This line is the best indication that now we are using spaces everywhere, no tabs.